test/Integration/test-csx/Common/utils.csx

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+#r "Newtonsoft.Json"
+
+using System;
+using System.Linq;
+using System.IO;
+using System.Text;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Serialization;
+
+public static class Utils
+{
+    /// <summary>
+    /// Default JSON serializer settings to use
+    /// </summary>
+    private static readonly JsonSerializerSettings _defaultJsonSerializationSettings;
+
+    static Utils()
+    {
+        _defaultJsonSerializationSettings = new JsonSerializerSettings
+        {
+            ContractResolver = new DefaultContractResolver()
+        };
+    }
+
+    /// <summary>
+    /// Serializes the specified object into a JSON string.
+    /// </summary>
+    /// <param name="obj">The object to serialize</param>
+    /// <param name="settings">The specific settings to use, uses a simple set of default settings if not specified</param>
+    /// <returns>The serialized JSON string</returns>
+    /// <remarks>This will NOT use any global settings to avoid picking up changes that may have been made by other code running in the host (such as user functions)</remarks>
+    public static string JsonSerializeObject(object obj, JsonSerializerSettings settings = null)
```

Comment:
Is there no way to import the existing Utils.cs file and use that? Avoids us having to remember to update both if we make changes there later on.